docs/cookbook/disabling-localised-urls.rst

```diff
@@ -0,0 +1,52 @@
+How to disable localised URLs?
+==============================
+
```

Comment:
We need some explanation on why would anyone need to disable these localised URLs?